doc/integrator/upgrade_application.rst.mako

```diff
@@ -25,6 +26,10 @@ Here an example that may apply to your situation:
 
 If you have no such managed files, define an empty section like this: ``managed_files: []``
 
+If you want that some files selected by expression in the ``default_project_file``, you can add them in an
```

Comment:
The way I understand it so far is:

If in your project you have files from the GeoMapFish templates which you did not customize, you can add these in a section ``unmanaged_files``. This will simplify the upgrade process for you, because the update script will then simply update these project files with the new GeoMapFish template files, and not list any changes for these in the "diff" file, therefore making it faster for you to review the differences during the upgrade process. The syntax for ``unmanaged_files`` is the same as for ``managed_files``.

Reply:
Thanks updated :-)